app/views/browse/build-config.html

```diff
@@ -288,7 +288,7 @@ <h4>{{imageSource.from | imageObjectRef : buildConfig.metadata.namespace}}</h4>
                               <div class="is-item-description">
                                 <dt>Paths:</dt>
                                 <div ng-repeat="(source, destination) in imageSourcesPaths[$index]" class="image-source-paths">
-                                  <dd><span class="source-path">{{source}}</span><i class="fa fa-long-arrow-right"></i><span class="destination-dir">{{destination}}</span></dd>
+                                  <dd><span class="source-path">{{source}}</span><i class="fa fa-long-arrow-right"></i><span class="secrets-location">{{destination}}</span></dd>
```

Comment:
Not new, but the icons should have `aria-hidden="true"`

Reply:
I don't think we should use the same class for the read-only view and the form inputs. They're different presentations and won't necessarily have the same padding.